setup.py

```diff
@@ -64,7 +65,7 @@ def RunCustomCommand(self, command_list):
       raise RuntimeError(
           'Command %s failed with error: %s' % (command_list, e)) from e
 
-  def run(self):
+  def do_install(self):
```

Comment:
This change is from #657 do we want to include it in this PR? I think it's better to keep these two issues separate? What do you think?